envs/genome.yaml

```diff
@@ -4,6 +4,8 @@ channels:
   - bioconda
   - defaults
 dependencies:
+  - python  >=3.9, <3.10.0a0
```

Comment:
I don't want to limit python version in "python" environment, as it should be main environment for more projects